server/src/controllers/decentralized_controller.ts

```diff
@@ -14,23 +14,22 @@ import MessageTypes = client.messages.type
 const debug = createDebug("server:controllers:decentralized")
 
 export class DecentralizedController extends TrainingController {
-  /**
-   * Set of nodes who have contributed.
-   */
-  private readyNodes = Set<client.NodeID>()
-  /**
-   * Map associating node ids to their open WebSocket connections.
-   */
-  private connections: Map<client.NodeID, WebSocket> = Map()
+  // Map of nodes who want to join the round.
+  // The boolean value indicates if the node is ready to exchange weight updates (i.e.
+  // the node has already sent a PeerIsRead message)
+  // We wait for all peers to be ready to exchange weight updates
+  #roundPeers = Map<client.NodeID, boolean>()
+  #aggregationRound = 0
```

Comment:
I was actually getting confused with the rounds because it's hard to know when using `round` if it is always the aggregation round or if we can infer from the context that it is the communication round (or even another one), so I would prefer expliciting the round type every time we use it and never use `round`. What do you think?

Reply:
yeah, good idea, explicit is better than implicit.
hopefully, we will get rid of this confusing "communication round" (keeping it internal to the aggregators).